Sources/BottomSheetView.swift

```diff
@@ -126,6 +126,11 @@ public final class BottomSheetView: UIView {
         addGestureRecognizer(panGesture)
 
         updateTargetOffsets()
+
+        let offset = BottomSheetCalculator.offset(for: contentView, in: superview, height: targetHeights[targetIndex])
```

Comment:
I think we actually don't need this. Here we call `transition(to:)`, which would call `animate` where `currentTargetOffsetIndex` will be set.

Reply:
We just need to make sure `transition(to:)` is called before `createTranslationTargets()` 👍